pilot/pkg/config/clusterregistry/clusterregistry.gotmpl

```diff
@@ -0,0 +1,16 @@
+apiVersion: clusterregistry.k8s.io/v1alpha1
+kind: {{.Kind}}
+metadata:
+  name: {{.Name}}
+  annotations:
+    config.istio.io/pilotEndpoint: "{{.PilotIP}}:9080"
+    config.istio.io/platform: "{{if .Platform}}{{.Platform}}{{else}}Kubernetes{{end}}"
+    {{if .PilotCfgStore}}config.istio.io/pilotCfgStore: "{{.PilotCfgStore}}"
```

Comment:
Same. If not required ( or has sensible defaults ) - remove, we can add it later if needed.

Reply:
Based on my limited understanding the reason behind is to be able to get "the central control pilot" quickly. When this struct is populated, the main control plane pilot will be annotated with it.

Reply:
I'm right there with you @costinm .  I think all of this is unnecessary - until the point at which multiple control planes are introduced.  Prior to the multicluster PR, the general thinking was multiple control planes.  Given exchanges on the ML, multiple Istio control planes (i.e. multiple plots) are a post 1.0 feature from my understanding.

That said, all of the validation code expects these annotations to be present, or pilot will fail to start with the loaded configmap.

Reply:
Ok, let's fix validation and simplify in separate PR.